quinn/src/connection.rs

```diff
@@ -85,6 +85,16 @@ where
             Err(self)
         }
     }
+
+    /// Data conveyed by the peer during the handshake, including cryptographic identity
```

Comment:
Should this have a documentation caveat about what the potential for this to be incomplete?

Reply:
Would `As the handshake is ongoing, the returned data is to be considered potentially incomplete.` do the job? I don't know much about all this, I just copied the inner comment.

Reply:
Maybe something like `Since the handshake is incomplete at this point, the returned data is likely to be incomplete as well.`?